File: output/ChatGPT/codenet/Java/C++/s505375619.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int n;
    cin >> n;
    vector<int> v(n);
    vector<int> c(n);
    vector<int> dp(n);

    for (int i = 0; i < n; i++) {
        dp[i] = 0;
    }
    for (int i = 0; i < n; i++) {
        cin >> v[i];
    }
    for (int i = 0; i < n; i++) {
        cin >> c[i];
    }

    dp[0] = 0;

    for (int i = 1; i < n; i++) {
        int tmp = dp[i - 1] + (v[i - 1] - c[i - 1]);
        dp[i] = tmp > dp[i - 1] ? tmp : dp[i - 1];
    }

    int tmp = dp[n - 1] + (v[n - 1] - c[n - 1]);
    dp[n - 1] = tmp > dp[n - 1] ? tmp : dp[n - 1];

    cout << dp[n - 1] << endl;

}

File: output/ChatGPT/codenet/Java/Python/s505375619.py
n = int(input())
v = list(map(int, input().split()))
c = list(map(int, input().split()))
dp = [0] * n

for i in range(1, n):
    tmp = dp[i - 1] + (v[i - 1] - c[i - 1])
    dp[i] = max(tmp, dp[i - 1])

tmp = dp[n - 1] + (v[n - 1] - c[n - 1])
dp[n - 1] = max(tmp, dp[n - 1])

print(dp[n - 1])